app/views/converts/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title "convert" %>
+<% content_for(:header) do %>
+  <h2>convert splits to other cool formats</h2>
+<% end %>
+<article>
+  <form id="convert" action="/api/v4/convert" method="post" enctype="multipart/form-data">
```

Comment:
Yes, the form is just a way to interface with the convert endpoint for humans.  I thought it would be cool to include it so that people could go and convert their runs to any format supported by splits-io if the timer they use doesn't have built in conversion features.